proto/relay.proto

```diff
@@ -42,6 +46,14 @@ message Ack {
   uint64 seq = 1;
 }
 
+message ConnectRequest {
```

Comment:
nit: for alignment I suggest either naming this to `EstablishConnectionRequest` or changing the `RelayService` method to `Connect`

Reply:
Hmm